prog/dftb+/lib_dftb/scc.F90

```diff
@@ -894,6 +910,7 @@ subroutine setShiftPerL(this, shift)
     @:ASSERT(size(shift,dim=1) == size(this%shiftPerL,dim=1))
     @:ASSERT(size(shift,dim=2) == size(this%shiftPerL,dim=2))
     
+    this%hasExternalShifts = .true.
```

Comment:
Same here. It should be a requirement for setting the shift rather than a consequence.